src/alto-ui/Form/Typeahead/Typeahead.js

```diff
@@ -173,13 +180,27 @@ const Typeahead = React.forwardRef(
 
     useEffect(() => setSearch(null), [isMenuOpen, value]);
 
+    const wasScrollbarOrInputClicked = () => {
```

Comment:
Move it out of this function - we don't have to create this function each time when component is reneder